homework.md

```diff
@@ -4,77 +4,78 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website prettier and neat
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 {
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+An object oriented programming language used for the functionality of web applications.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+-It tracks changes you make to a file so you have a record of your history
+-It makes collaborating easy
+-It's a central place where you can store, upload and download file changes
+
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You create a copy of that repository so that when you make changes you don't affect the original project.
```

Comment:
Hi Xolani, firstly well done on completing your homework. I think that part of the answer is where the repository gets copied to.

Reply:
Hey Chris,

I actually learnt the answer from github docs, here's a link: https://docs.github.com/en/get-started/quickstart/fork-a-repo

And i do get your point but i think the answer is self explanatory and it actually answers the question 🤷🏾‍♂️ though I'm sorry it doesn't meet your standard😅 but it  doesn't make it incorrect either.

Thank you.